web/stats/dlc_structures.php

```diff
@@ -342,9 +339,9 @@
       "los": "20",
       "hp": "2200",
       "ra": "7",
-      "at": "6/arrow",
+      "at": "6+ 3x 1",
```

Comment:
This one is fun, and confusing. All over chu-ko-nus again. I should've looked into this in more detail.

Looking into it again. There are 4 arrows. The first one should be:
6 pierce damage + 9 against ships
the secondary arrows have +1 against ships and +1 against camels on them. So this should add up 12 against ships, not 13. I got to watch zero's chu-ko-nu video again to make heads and tails of this. Also those extra arrows should do at minimum 1 damage. I forgot how that combines with bonuses.